docs/lessons/lesson1-10/Boltdir/site-modules/exercise7/tasks/write.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+if [ -z "$PT_message" ]; then
```

Comment:
It'd be nice to have this be consistent with exercise 8, and I like calling it 'content' better than message if we're writing it to a file. Might need to update the lesson too